install/on_rpi/on_pi_setup.sh

```diff
@@ -40,6 +40,8 @@ sudo cp $HOME/bthidhub/install/on_rpi/sdp_record.xml /etc/bluetooth/sdp_record.x
 sudo cp $HOME/bthidhub/install/on_rpi/input.conf /etc/bluetooth/input.conf
 sudo cp $HOME/bthidhub/install/on_rpi/main.conf /etc/bluetooth/main.conf
 
+sudo sed -i "s/\/home\/pi/\/home\/$(basename $HOME)/g" $HOME/bthidhub/install/on_rpi/remapper.service
+sudo sed -i "s/PI_USER = 'pi'/PI_USER = '$(basename $HOME)'/g" $HOME/bthidhub/web.py
```

Comment:
Same with uncommitted change in this file. At worst, we could write a new file and have the code read the user from that file. Maybe a better approach would be to pass the user in the service file to the application?